### PR TITLE
[Tests-Only] Change storage driver from `owncloud` to `ocis` for webUITests pipeline

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Changes storage driver from `owncloud` to `ocis` for webUI tests pipeline

## Related Issue
- Closes https://github.com/owncloud/ocis/issues/1987

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by phil-davis** [comment]
What should we do about https://github.com/owncloud/ocis/blob/master/tests/acceptance/expected-failures-webUI-on-OWNCLOUD-storage.md ?

If we are not running CI any more with OWNCLOUD storage, then the listof scenarios in there will get out-of-date.

Do we delete it? Or?

**Comment by saw-jan** [comment]
> What should we do about https://github.com/owncloud/ocis/blob/master/tests/acceptance/expected-failures-webUI-on-OWNCLOUD-storage.md ?
> 
> If we are not running CI any more with OWNCLOUD storage, then the listof scenarios in there will get out-of-date.
> 
> Do we delete it? Or?

we can delete it if we won't run tests against owncloud storage in upcoming days. Or other thoughts?
@individual-it @phil-davis

**Comment by individual-it** [comment]
yes, I would propose to delete that file

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 1]
```diff
@@ -57,10 +57,10 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIRenameFiles/renameFiles.feature:227](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIRenameFiles/renameFiles.feature#L227)
 -   [webUIRenameFiles/renameFiles.feature:249](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIRenameFiles/renameFiles.feature#L249)
 -   [webUISharingInternalUsers/shareWithUsers.feature:57](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L57)
--   [webUISharingInternalUsers/shareWithUsers.feature:58](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L58)
+<!-- -   [webUISharingInternalUsers/shareWithUsers.feature:58](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L58) -->
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:60]: There are bonus `<!--` `-->` - is that an accident?

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 2]
```diff
@@ -318,7 +314,6 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:236](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L236)
 
 ### [renamed file cannot be shared](https://github.com/owncloud/web/issues/4192)
--   [webUISharingInternalUsers/shareWithUsers.feature:341](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L341)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:321]: If this is the only failure for issue 4192, then the heading could be deleted also.

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 3]
```diff
@@ -295,13 +292,12 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
--   [webUITrashbinDelete/trashbinDelete.feature:99](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L99)
+-   [webUITrashbinDelete/trashbinDelete.feature:50](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L50)
+-   [webUITrashbinDelete/trashbinDelete.feature:86](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L86)
+-   [webUITrashbinDelete/trashbinDelete.feature:106](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L106)
 
 ### [Propfind response to trashbin endpoint is different in ocis](https://github.com/owncloud/product/issues/186)
 ### [restoring a file from "Deleted files" (trashbin) is not possible if the original folder does not exist any-more](https://github.com/owncloud/web/issues/1753)
--   [webUITrashbinRestore/trashbinRestore.feature:138](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinRestore/trashbinRestore.feature#L138)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:304]: If this is the only failure for issue 186 and 1753, then the heading could be deleted also.

**Comment by kiranparajuli589** [comment]
rather than exisiting comments..lgtm 👍🏼

**Review by kiranparajuli589 — COMMENTED** [review]
rather than the exisiting comments..lgtm 👍🏼

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 4]
```diff
@@ -295,20 +292,18 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
 
 ### [Conflict / overwrite issues with TUS](https://github.com/owncloud/ocis/issues/1294)
 -   [webUIUpload/uploadEdgecases.feature:109](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/uploadEdgecases.feature#L109)
 -   [webUIUpload/uploadFileGreaterThanQuotaSize.feature:12](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/uploadFileGreaterThanQuotaSize.feature#L12)
 
 ### [restoring a file deleted from a received shared folder is not possible](https://github.com/owncloud/ocis/issues/1124)
--   [webUITrashbinRestore/trashbinRestore.feature:260](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinRestore/trashbinRestore.feature#L260)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:311]: If this in the only failure for issue 1124 then the heading can also be deleted.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/4808/69/7 had a webUI test fail.
"Unexpected failure or crash of the nightwatch test run"
I restarted CI.

**Review by jasson99 — COMMENTED** [review]
LGTM

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/4813/69/7
"Unexpected failure or crash of the nightwatch test run"

That is happening every time I restart, but only for `webUITrashbinRestore` suite.

What is going on?

**Comment by saw-jan** [comment]
That one will always fail. I think the problematic scenario is `trashbinRestore.feature:260` and it is tagged with issue `https://github.com/owncloud/ocis/issues/1128`
It happens when `ocis` storage previously with `owncloud` storage it was working fine (I mean was failing fine)

**Comment by phil-davis** [comment]
We need to think what to do. It is strange that we can make a crash like this in the test scripts. There is no way to add this to expected-failures! So next week we need to sort out how to fix it.

**Comment by saw-jan** [comment]
> We need to think what to do. It is strange that we can make a crash like this in the test scripts. There is no way to add this to expected-failures! So next week we need to sort out how to fix it.

Should be fixed by PR https://github.com/owncloud/web/pull/5061

**Review by phil-davis — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**